src/components/magasin/grid/LeftMenu.js

```diff
@@ -33,7 +33,7 @@ export default class NodeLeftMenuComponent extends Component {
       showButtons
     } = this.props;
 
-    const buttonLink = (type, icon, eventType, MusitIconType) => {
+    const buttonLink = (type, icon, eventType, MusitIconType, disabled) => {
```

Comment:
move disabled before MusitIconType and rename MusitIconType useMusitIcon and change from sending 1 to true